docs/get-started/run.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_label: 'Run the transformation pipeline'
+sidebar_position: 6
+---
+
+# Run the transformation pipeline
+
+In the previous steps of this tutorial we spent some time to create necessary components and connect
+them all together to form our pipeline DAG. The first step before getting to a proper running pipeline
+is to make sure that the DAG we created id valid.
+
+To do so the sub-command `holium project export --no-write` is available:
+```shell
+$ holium project export --no-write
+current local project holds a healthy transformation pipelin
+```
+
+Now that we know that our pipeline DAG is sane we will focus on the execution.
+
+The only thing left to do before running our transformation pipeline is to define **portations** to
+import data to Holium and export data from Holium. In our case we will apply two import **portations**
+to our sources and one export **portation** to our shaper.
+
+The different files that we will use as our imported data can be found [in our dedicated repository](https://github.com/polyphene/getting-started/tree/main/assets/wasm).
```

Comment:
Looks like a wrong link to `wasm` directory.

Reply:
The aim was to link to the `./assets/wasm` folder in the repository _getting-started_. It will actually direct to `main` once the repository is merged. The current link for opened branch / PR would be https://github.com/polyphene/getting-started/tree/feature/tutorial-repository/assets/wasm

Reply:
I understand, but why not keeping it at its default location, that is `./module/avocado_operations/target/wasm32-unknown-unknown/release/avocado_operations.wasm`?